lib/components/edit-bundle.tsx

```diff
@@ -1,46 +1,161 @@
-import Select from './select'
+import {ChevronDown, ChevronUp, DeleteIcon} from './icons'
+import LabelHeading from './label-heading'
+import IconButton from './icon-button'
 
-const getOptionLabel = (b) => b.name
+const hasContent = (s: string) => s.length > 0
 
-const hasContent = (s) => s.length > 0
+function getSummaryStatus(
+  summary: CL.GTFSErrorTypeSummary
+): AlertProps['status'] {
+  switch (summary.priority) {
+    case 'HIGH':
+      return 'error'
+    case 'MEDIUM':
+      return 'warning'
+    case 'LOW':
+    default:
+      return 'info'
+  }
+}
+
+function formatType(errorType: string) {
+  return startCase(errorType) + 's'
+}
+
+function ErrorType({error}: {error: CL.GTFSErrorTypeSummary}) {
+  const {isOpen, onToggle} = useDisclosure({defaultIsOpen: true})
+  const name = formatType(error.type)
+  return (
+    <Alert
+      alignItems='flex-start'
+      flexDir='column'
+      status={getSummaryStatus(error)}
+    >
+      <Flex mb={isOpen ? 2 : 0} width='100%'>
+        <AlertIcon />
+        <AlertTitle flex='2' fontSize='xl'>
+          {name} ({error.count})
+        </AlertTitle>
+        <IconButton
+          onClick={onToggle}
+          label={isOpen ? `Hide ${name}` : `Show ${name}`}
+          position='absolute'
+          top='8px'
+          right='8px'
+        >
+          {isOpen ? <ChevronUp /> : <ChevronDown />}
+        </IconButton>
+      </Flex>
+
+      <Box width='100%' mb={4}>
+        <Collapse in={isOpen}>
+          <Stack spacing={6}>
+            {error.someErrors.map((errorSummary, index) => (
+              <HStack key={index} spacing={4} justify='space-between'>
+                <LabelHeading>#{index + 1}</LabelHeading>
+                <Stack flex='1' spacing={0}>
+                  <LabelHeading>file</LabelHeading>
+                  <Heading size='md'>{errorSummary.file}.txt</Heading>
+                </Stack>
+                {errorSummary.line != null && (
+                  <Stack flex='1' spacing={0}>
+                    <LabelHeading>line</LabelHeading>
+                    <Heading size='md'>{errorSummary.line}</Heading>
+                  </Stack>
+                )}
+                {errorSummary.field != null && (
+                  <Stack flex='1' spacing={0}>
+                    <LabelHeading>field</LabelHeading>
+                    <Heading size='md'>{errorSummary.field}</Heading>
+                  </Stack>
+                )}
+                <Stack flex='2' spacing={0}>
+                  <LabelHeading>message</LabelHeading>
+                  <Heading size='md'>{errorSummary.message}</Heading>
+                </Stack>
+              </HStack>
+            ))}
+          </Stack>
+        </Collapse>
+      </Box>
 
-function FeedNameInput({feed, index, onChange, ...p}) {
+      {error.count > 10 && (
+        <Text textAlign='center' fontStyle='italic' mt={2} width='100%'>
+          Note: only the first ten errors of each type are saved.
```

Comment:
```suggestion
          plus {error.count - 10} more errors of this type.
```

Also suggest moving this into the collapse above -- shouldn't need to be shown when boxes are collapsed.